src/components/calculator.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 // eslint-disable-next-line
 import RenderCalculator from './functions/renderCalculator';
 
-const Calculator = () => (
-  <div className="grid">
-    <form className="form">
-      <input className="number" type="number" placeholder="0" />
-    </form>
-    <RenderCalculator />
-  </div>
-);
+const Calculator = () => {
```

Comment:
- Hi, I would advise you not to disable the linters because is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Also, the requirement of the project prohibits disabling the linter
- Please, check if your App is working fine before asking for re-review
- Readme more about [Linters](https://github.com/microverseinc/linters-config)

Reply:
Hi @OLIPLICHE and the next reviewer 👋🏼 
The last PR that I did. I had a problem with the linters. The problem is that the linters run well in my Visual Studio without issues, but GitHub shows linters issues with the file extensions. I was trying with my last CR to fix that weird error in different ways but nothing works. So my CR told me that only put the "linters disabled" because the error couldn't be fixed. 

I would appreciate that understand this situation and allow me to put those lines to fix that issue.

Thanks in advance 🙌🏼